package.json

```diff
@@ -71,16 +77,19 @@
     "html-webpack-plugin": "^2.30.0",
     "json-loader": "^0.5.4",
     "nightwatch": "^0.9.12",
+    "node-sass": "^4.5.2",
+    "postcss-loader": "^2.0.6",
+    "raw-loader": "^0.5.1",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
     "rimraf": "^2.6.1",
-    "webpack": "^3.6.0",
-    "webpack-dev-server": "^2.9.1",
-    "node-sass": "^4.5.2",
-    "postcss-loader": "^2.0.6",
+    "sass-loader": "^6.0.6",
     "shelljs": "^0.7.6",
     "style-loader": "^0.19.0",
-    "raw-loader": "^0.5.1",
-    "sass-loader": "^6.0.6"
+    "wdio-mocha-framework": "^0.5.11",
```

Comment:
If we want to recommend these 'wdio' items, I suggest adding them as dependencies so that they get transitively pulled in.